.env.example

```
@@ -0,0 +1,22 @@
+# Social Media API Keys
+TWITTER_BEARER_TOKEN=your_twitter_bearer_token_here
+TWITTER_API_KEY=your_twitter_api_key_here
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The TWITTER_API_KEY key should go before the TWITTER_BEARER_TOKEN key

(UnorderedKey)